src/sentry/issues/auto_source_code_config/constants.py

```diff
@@ -11,3 +11,5 @@
 ]
 # These languages will run as dry-run mode by default
 DRY_RUN_PLATFORMS: list[str] = []
+# Some languages will also process system frames
+PROCESS_ALL_FRAMES: list[str] = []
```

Comment:
We may one day do this for all platforms, however, for now, we will keep to a short list.